polaris-react/src/components/Tag/Tag.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import React, {useCallback, useState} from 'react';
 import type {ComponentMeta} from '@storybook/react';
-import {InlineStack, Icon, LegacyStack, Tag} from '@shopify/polaris';
-import {Wand} from '@shopify/polaris-icons';
+import {InlineStack, Icon, LegacyStack, Tag, Bleed} from '@shopify/polaris';
+import {MagicMinor, Wand} from '@shopify/polaris-icons';
```

Comment:
I think `polaris-icons` has been updated to no longer have `Minor` or `Major` anymore.

```suggestion
import {Magic} from '@shopify/polaris-icons';
```